tests/framework/ensembleModelTests/test_ensemble_model_2_codes_optional_output.xml

```diff
@@ -6,14 +6,15 @@
     <created>2017-05-17</created>
     <classesTested>Models.Code, JobHandler.Thread</classesTested>
     <description>
-       The main goal of this test is to show how to specify optional output in the Ensemle Model object.
+       The main goal of this test is to show how to specify optional output in the Ensemble Model object.
+       All the RAVEN available data storage types are tested (DataObjects and Databases).
     </description>
```

Comment:
I agree. This extra bit of information is used by the automatic regression tests documentation. We probably could just create a script that gets the log for each file and add it directly in the latex.

What do you think?